game_event/models.py

```diff
@@ -10,7 +10,6 @@
 
 
 class GameEvent(models.Model):
-    id = models.IntegerField(primary_key=True)
     time = models.DateTimeField(null=False)
```

Comment:
Can you explain me why this line was deleted? 
`id = models.IntegerField(primary_key=True)`